Packs/PaloAltoNetworks_SecurityAdvisories/Integrations/PaloAltoNetworksSecurityAdvisories/command_examples.txt

```diff
@@ -1 +1,2 @@
 !pan-advisories-get-advisories product="PAN-OS" q=CVE-2022-0778
+!pan-advisories-get-advisories product="PAN-OS" q=CVE-2023-38802
```

Comment:
In the `command_example.txt` we used to write only one example per command, so please delete line no.2
🙏

Reply:
removed